Add support for /entity/view/ paths in uuid resolver

elbotho opened the pull request:
(closes #143)

kulla commented on src/graphql/schema/uuid/abstract-uuid/resolvers.ts:
```diff
@@ -53,6 +53,17 @@ export const resolvers: AbstractUuidResolvers = {
         }
+
+        //TODO: to support legacy feature. remove after frontend is 100% (#143)
+        if (cleanPath.startsWith('/entity/view/')) {
+          const match = /^\/entity\/view\/(\d+)$/.exec(cleanPath)
+          if (match) {
+            const id = parseInt(match[1], 10)
+            const uuid = await dataSources.serlo.getUuid<UuidPayload>({ id })
+            return uuid || null
+          }
+        }
```
```suggestion
          const match = /^\/entity\/view\/(\d+)$/.exec(cleanPath)
          if (match) {
            const id = parseInt(match[1], 10)
            return await dataSources.serlo.getUuid<UuidPayload>({ id })
          }
        
```

First `if` statemenent is redundant

elbotho replied:
just went along with the profile example above, thought it's some tiny performance gain :)
Will change it above then as well, right?

kulla reviewed (changes requested):
Can you add your code to https://github.com/serlo/api.serlo.org/pull/140/files (or start a new PR merging into this PR)? -> Then we have no merge conflicts :-) Tests would be great.

kulla commented:
Here how you can add a test: https://github.com/serlo/api.serlo.org/blob/alias-followups/__tests__/schema/uuid/user.ts#L56-L84 However I would add a test to `__tests__/schema/uuid/abstract-uuid.ts`.

elbotho commented:
added tests, simplified `cleanPath` checks. 
not sure if it's a good idea to PR onto a PR based on another PR at this point or if we should just wait until the other ones are merged and rebase?

inyono commented:
Rebased onto #157.

inyono reviewed (approved):
Requested changes seem to be addressed.

elbotho commented:
Thanks @inyono !
@kulla or @CarolinJaser please ping me if I can help resolve merge conflicts should they come up.
